data/mods/innawood/itemgroups/guns.json

```diff
@@ -262,13 +262,13 @@
   {
     "type": "item_group",
     "id": "sidearms_cop",
-    "items": [ { "group": "guns_pistol_improvised", "prob": 20, "charges": [ 0, 1 ] } ]
+    "items": [ { "group": "null", "prob": 20 } ]
```

Comment:
No flintlock pistol?

Reply:
Tried, but it was throwing pocket errors. This group is for going into holsters that don't fit a long flintlock pistol.